.github/workflows/dist.yml

```
@@ -73,8 +73,14 @@ jobs:
         uses: hendrikmuhs/ccache-action@v1
         with:
           create-symlink: true
-          key: ccache-${{ runner.os }}-${{ runner.arch }}
           evict-old-files: "7d"
+          append-timestamp: false
+          key: ccache-sdist-${{ runner.os }}-${{ runner.arch }}-${{ matrix.target.toolkit }}
```

actionlint (1.7.8)

property "target" is not defined in object type {}

(expression)